src/App.tsx

```diff
@@ -1,8 +1,14 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goods = [
+type Good = string;
+type State = {
+  selectedGood: Good | '';
```

Comment:
it's like string | string

Reply:
![image](https://user-images.githubusercontent.com/79811705/197065856-3e6f5b7b-a2a2-4c9d-8226-78bffc53e6eb.png)

our coach recommended to write like this

Reply:
So, string | string - it's ok?